src/main/java/com/autilus/hw2806/MailService.java

```diff
@@ -0,0 +1,31 @@
+package com.autilus.hw2806;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+public static class MailService<T> implements Consumer<Send<T>> {
+    @Override
+    public void accept(Send<T> t) {
+        List<T> v = box.get(t.getTo());
+        v.add(t.getContent());
+        box.put(t.getTo(), v);
+    }
+
+    public MailService() {
+        box = new HashMap<String, List<T>>() {
+            @Override
+            public List<T> get(Object key) {
```

Comment:
use lambda

Reply:
empty list, use getOrDefault method

Reply:
when we use Collections.emptyList() we need to change method accept. But, we can leave everything as it is and use new LinkedList().